keyboards/redox/keymaps/media-ch/config.h

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2021 Shiftux <shiftux@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+/* Use I2C or Serial, not both */
+#define USE_SERIAL
+// #define USE_I2C
+
+/* Select hand configuration */
+#define MASTER_LEFT
+// #define MASTER_RIGHT
+#define EE_HANDS
```

Comment:
This should be one or the other, not both.  Though ... ee hands will take precedence here.

Reply:
i'm a bit confused about that statement. i thought EE_HANDS would specify that there is a specific eeprom-flashed piece of code which lets QMK know that it would not have to auto-detect which split part is which and which would be the master. however there still needs to be a master, no? - I mean, i only have 1 USB cable connected and then a serial connection to the other hand. - so how is the master set?

Reply:
Nope.  EE_HANDS tells it to look to the EEPROM to determine which side is left or right. 

MASTER_LEFT and MASTER_RIGHT tells the firmware that the master side (the side the USB cable is in) is the left or right side, respectively.   So you only want one of these options defined.